.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+pipeline {
+	agent any
+	options {
+		lock("homeworld")
+	}
+	stages {
+		stage('Build') {
+			agent {
+				docker {
+					image 'debian:stretch-slim'
```

Comment:
Is... is this image being pulled from the docker registry? (I'm generally a bit sketched out by docker's distribution model.)

Is there any alternative to pulling down an image from the public registry?

Reply:
For reference, we decided to leave this issue for later since Docker doesn't currently seem to provide a way to verify images. I have verified that the image is only downloaded once (later launches use the local copy) and that the image is from an Official Repository (vetted by Docker).